src/index.jsx

```diff
@@ -17,7 +28,29 @@ import configureStore from './data/configureStore';
 import './index.scss';
 import './assets/favicon.ico';
 
-App.subscribe(APP_READY, () => {
+mergeConfig({
+  CURRENCY_COOKIE_NAME: process.env.CURRENCY_COOKIE_NAME,
+  SUPPORT_URL: process.env.SUPPORT_URL,
+  CYBERSOURCE_URL: process.env.CYBERSOURCE_URL,
+  APPLE_PAY_MERCHANT_NAME: process.env.APPLE_PAY_MERCHANT_NAME,
+  APPLE_PAY_COUNTRY_CODE: process.env.APPLE_PAY_COUNTRY_CODE,
+  APPLE_PAY_CURRENCY_CODE: process.env.APPLE_PAY_CURRENCY_CODE,
+  APPLE_PAY_START_SESSION_URL: process.env.APPLE_PAY_START_SESSION_URL,
+  APPLE_PAY_AUTHORIZE_URL: process.env.APPLE_PAY_AUTHORIZE_URL,
+  APPLE_PAY_SUPPORTED_NETWORKS: process.env.APPLE_PAY_SUPPORTED_NETWORKS && process.env.APPLE_PAY_SUPPORTED_NETWORKS.split(','),
+  APPLE_PAY_MERCHANT_CAPABILITIES: process.env.APPLE_PAY_MERCHANT_CAPABILITIES && process.env.APPLE_PAY_MERCHANT_CAPABILITIES.split(','),
+});
+
+subscribe(APP_ANALYTICS_INITIALIZED, () => {
```

Comment:
We're  putting this code into an analytics-phase handler in frontend-platform, so this whole subscribe block should be deletable.

Reply:
Swapped